src/tool/hpcrun/scripts/hpcrun.in

```diff
@@ -520,9 +536,9 @@ if test ! -x "$HPCRUN_FNBOUNDS_CMD" ; then
 fi
 export HPCRUN_FNBOUNDS_CMD
 
-export LD_LIBRARY_PATH="${hpc_ld_library_path}:${LD_LIBRARY_PATH}"
+export LD_LIBRARY_PATH="${hpc_ld_library_path}${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}"
 export LD_PRELOAD="${preload_list} ${LD_PRELOAD}"
-@export_ld_audit@
+export LD_AUDIT="${audit_list}:${LD_AUDIT}"
```

Comment:
What if either of these terms is NULL? Is the leading or trailing : something that should be avoided? Do we even want to set LD_AUDIT if both LD_AUDIT and audit_list are empty?

Reply:
Acts as a no-op, but fixed anyway.